src/components/Slider/Slider.svelte

```diff
@@ -15,6 +15,9 @@
 
   export let classes = classesDefault;
 
+  $: percentFactor = 100.0 / (max - min);
+
+  $: toPercent = v => (v - min) * percentFactor;
```

Comment:
Why not put all calculations in the same reactive statement?

Reply:
Hmm... sounds reasonable.
I'm quite newbie in svelte.
Will it cache calculation of factor?

Reply:
Both will change if `min` changes.
```
 $: toPercent = v => (v - min) * (100.0 / (max - min));
```
should work ok.

Reply:
ok